lib/instrumentation/instrumentor.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+var utils = require('../utils');
+
+var defaultOnConfig = {
+  console: true
+};
+
+var defaultConfig = {
+  console: false,
+  http: false,
+  pg: false
+};
+
+function fill(obj, name, replacement, track) {
```

Comment:
Yea, sure, I had a mental note on this but forgot about it. This was another of my motivating factors for a shared js utils, but that's a separate issue.